util/test/tests/Vulkan/VK_Overlay_Test.py

```diff
@@ -91,4 +120,6 @@ def check_capture(self):
 
         rdtest.log.success("Reference and output image are identical for stencil")
 
+        self.check_clearbeforedraw_depth(out, pipe.GetDepthTarget().resourceId)
```

Comment:
I should definitely deduplicate these files like I did with the Texture_Zoo ones, since they're identical other than the test name. Don't worry about doing that in this PR though.